api/src/main.rs

```diff
@@ -0,0 +1,42 @@
+#[macro_use]
+extern crate rocket;
+use sqlx::postgres::PgPool;
+use sqlx::postgres::PgPoolOptions;
+
+#[get("/")]
+async fn index(pool: &rocket::State<PgPool>) -> String {
+    // Make a simple query to return the given parameter (use a question mark `?` instead of `$1` for MySQL)
+    let test_id: i32 = match sqlx::query!("SELECT id FROM test")
+        .fetch_one(pool.inner())
+        .await
+    {
+        Ok(test_id) => test_id.id,
+        Err(e) => {
+            // One day we should handle errors better (HTTP return codes)
```

Comment:
Investigate bubbling up errors using `?`, see what rocket likes

Reply:
Yeah, played with this a lot. It's non-obvious how to elegantly marshall Rust Errors into HTTP Error codes. I'll leave this for now, but plan on doing more here soon.